libs/wingc/src/lsp/signature.rs

```diff
@@ -177,10 +208,14 @@ pub struct ScopeVisitor<'a> {
 	pub location: Position,
 	/// The nearest expression before (or containing) the target location
 	pub call_expr: Option<&'a Expr>,
+	/// The nearest statement before (or containing) the target location - on a case we're on a super() call
+	pub call_stmt: Option<&'a Stmt>,
 	// The env of the found expression
 	pub call_env: Option<SymbolEnvRef>,
 	/// The current symbol env we're in
 	curr_env: Vec<SymbolEnvRef>,
+	/// the nearest class containing the call_stmt
+	pub class: Option<&'a Class>,
```

Comment:
It should be possible to avoid storing this, the `call_env` will be a SymbolEnv whose parent will be the class environment.

So you should be able to use `call_env?.parent?.lookup`

Reply:
I tried it with call_env being assigned when the call_stmt is assigned and when the class is assigned, but none of them worked unfortunately :(